core/file_monitor.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import time
+from watchdog.observers import Observer
+from watchdog.events import FileSystemEventHandler
+from database.connector import insert_file_change_events
+from core.alert import info
+from core.compatible import byte_to_str
+from core.time_helper import now
+
+
+def is_excluded(path, dirs):
+    """
+    if path is excluded in list of dirs/files
+
+    :param path: path to check for exclude
+    :param dirs: list of excludes
+    :return: Boolean
+    """
+    for directory in dirs:
+        if path.startswith(directory):
+            return True
+    return False
+
+
+class containerFilesHandler(FileSystemEventHandler):
+    def __init__(self):
+        self.log_filename = None
+        self.EXCLUDES = None
+        self.module_name = None
+
+
+    def on_any_event(self,event):
+        if not (event.event_type == 'modified' and event.is_directory) and is_excluded(event.src_path, self.EXCLUDES):
+            insert_file_change_events(
+                byte_to_str(event.src_path),
+                byte_to_str(event.event_type),
+                self.module_name,
+                now()
+            )
+            info("Event on a file: " + byte_to_str(event.event_type) + " , path: " + byte_to_str(event.src_path))
+
+
+class fileMonitor:
```

Comment:
class names should be CapWords convention.
https://www.python.org/dev/peps/pep-0008/#class-names